httpd/handler.go

```diff
@@ -175,6 +179,127 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *influ
 	httpResults(w, results, pretty)
 }
 
+type Point struct {
```

Comment:
omitempty appears not to work for struct fields of type time.Time. I can't reuse client.BatchPoints here. I can reuse client.Point here if you really want me to. I'm not sure it's a good idea though.